src/addAndDelete.test.js

```diff
@@ -0,0 +1,18 @@
+import add from './modules/add.js';
+import remove from './modules/remove.js';
+
+describe('Add test', () => {
+  test('should save to localStorage', () => {
+    localStorage.setItem('tasks', '[]');
+    add('hello');
+    expect(JSON.parse(localStorage.getItem('tasks'))).toEqual([{ completed: false, description: 'hello', index: 1 }]);
+  });
+});
+
+describe('Remove test', () => {
+  test('should remove from localstorage', () => {
+    localStorage.setItem('tasks', JSON.stringify([{ completed: false, description: 'task1', index: 1 }]));
+    remove('task1');
+    expect(JSON.parse(localStorage.getItem('tasks'))).toEqual([]);
+  });
+});
```

Comment:
- [ ] Great work testing your add and remove functions. When I run the `npm run test` command, I get an error that `Jest-environment-dom` can not be found. You need to include `"jest-environment-jsdom": "^29.4.3"` in your `package.json` file or advise those who clone your repository to run `npm install -D jest-environment-jsdom` after running `npm install` in your _**README.md**_ file. 👍 😎 

![image](https://user-images.githubusercontent.com/77106962/222269714-d1e67c0c-6ba9-45d2-81a2-be9330b3c7a1.png)

Reply:
Hi @Gambit142 , Ok! we added it to the package.json. Thank you for the review, now we are going to request another one. 

![image](https://user-images.githubusercontent.com/118059775/222271514-9140a2ac-c1bb-4a8a-8cee-d178c37a8b0f.png)